pages/jobs.tsx

```diff
@@ -144,7 +144,12 @@ const Jobs: NextPage<JobsPageProps> = ({ preview, jobs, content }) => {
                         secondLine={content.secondTitleLine}
                     />
                     <div className={styles.textContainer}>
-                        <RichText theme="INDIGO" content={content.subtitle} />
+                        {content.subtitle && (
+                            <RichText
+                                theme="INDIGO"
+                                content={content.subtitle}
+                            />
+                        )}
```

Comment:
We've remove the link to learn more about the application process, meaning this section is now required to be conditional.